kuwala/core/cli/src/InputController.py

```diff
@@ -50,6 +50,7 @@ def select_region(pipelines: [str]) -> [str, str]:
     country_region = None
     population_density_id = None
     osm_url = None
+    update_date= None
```

Comment:
I feel the variable name is ambiguous right now. `population_density_update_date` gives more context since the variables above are not only referring to one pipeline.